eslint-plugin-expensify/tests/no-object-keys-includes.test.js

```diff
@@ -0,0 +1,187 @@
+import {RuleTester} from '@typescript-eslint/rule-tester';
+import {fileURLToPath} from 'url';
+import parser from '@typescript-eslint/parser';
+import path from 'path';
+import * as rule from '../no-object-keys-includes.js';
+import CONST from '../CONST.js';
+
+const filename = fileURLToPath(import.meta.url);
+const dirname = path.dirname(filename);
+const tsconfigRootDir = path.resolve(dirname, '../fixtures');
+
+const message = CONST.MESSAGE.NO_OBJECT_KEYS_INCLUDES;
+
+const ruleTester = new RuleTester({
+    languageOptions: {
+        parser,
+        parserOptions: {
+            project: './tsconfig.json',
+            tsconfigRootDir,
+            sourceType: 'module',
+            ecmaVersion: 2020,
```

Comment:
NAB, let's use 2024?


```suggestion
            ecmaVersion: 2024,
```